inference-engine/tests/ie_test_utils/unit_test_utils/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ add_subdirectory(mocks/mock_engine)
 list(APPEND EXPORT_DEPENDENCIES
         commonTestUtils_s
         inference_engine_s
-        inference_engine_lp_transformations
+        inference_engine_lp_transformations_s
```

Comment:
may be we don't need LP transformation to test utils at all? Looks strange that utils depends on LP transformations

Reply:
Static version of this library still might be useful for plugins unit tests (if some plugin use this library and want to test functionality based on it).

Reply:
I think test which use LP transformations can be functional, not unit ones. I don't want to compile IR libraries twice..

Reply:
> This transformations are done on IR level and can be tested by unit test

For such cases we have `ieFuncTests`, can we put tests for transformations there?

Reply:
> can we put tests for transformations there?

I meant internal plugin transformations that uses some stuff from this library.

> testing of static version is not the same as testing of dynamic one

I suppose this is one of the key difference between unit and functional tests. Unit tests checks internal code and can work with static version, while functional tests perform white box testing using public API against dynamic libraries. So, unit test can check some hidden code that is not exported in dynamic libraries.

Reply:
> I meant internal plugin transformations that uses some stuff from this library.

Internal plugin transformations can be tested via plugin_static library. Why do we need LP transformations compiled as static?

> I suppose this is one of the key difference between unit and functional tests

Even internal / hidden pieces of code can be tested via high level API. E.g. create such flow, which trigger a testing function to run.

BTW, do we have any strict reason to call some LP transformation internals right now? This library is going to be rewritten to ngraph, and I suppose all current code will be thrown away. So, no needs to introduce new dependencies on old code in newly added tests, is not it?